goblint-svcomp/run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+shopt -s extglob
+set -e
+
+BENCH_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+SV_BENCHMARKS_DIR=/mnt/goblint-svcomp/benchexec/sv-benchmarks
+RESULTS_DIR=/mnt/goblint-svcomp/benchexec/results/142-concurrency-witnesses
+GOBLINT_DIR=/mnt/goblint-svcomp/sv-comp/goblint
+GOBLINT_PARALLEL=2
+GOBLINT_VALIDATE_PARALLEL=4
+
+# read-only and overlay dirs for Value too large for defined data type workaround
+BENCHEXEC="benchexec --read-only-dir / --overlay-dir . --overlay-dir /home --outputpath $RESULTS_DIR"
+
+mkdir $RESULTS_DIR || true
+
+# Construct verification XML
+cd $RESULTS_DIR
+sed -e "s|SV_BENCHMARKS_DIR|$SV_BENCHMARKS_DIR|" $BENCH_DIR/goblint.xml > goblint.xml
```

Comment:
This is a welcome improvement, means we can use the script too here in Munich :smile: